src/spetlr/delta/db_handle.py

```diff
@@ -28,8 +28,8 @@ def from_tc(cls, id: str):
         tc = Configurator()
         return cls(
             name=tc.table_name(id),
```

Comment:
What is the difference between `table_id` and `table_name`? I presume both uniquely identifies the table, and hence why not use the `table_name` as the key?

Reply:
There is a huge difference. We need to write better documentation, but maybe not in a PR review comment. Maybe make an issue to ask for better explanation?